climpred/prediction.py

```diff
@@ -100,16 +116,20 @@ def _get_metric_comparison_dim(metric, comparison, dim, kind):
         comparison (Comparison): comparison class.
         dim (list of str or str): corrected dimension to apply metric to.
     """
+    dim = _sanitize(dim)
+
     # check kind allowed
     is_in_list(kind, ['hindcast', 'PM'], 'kind')
-    # set default dim
+    # set default dim # TODO: should we still do this?
     if dim is None:
-        dim = 'init' if kind == 'hindcast' else ['init', 'member']
-    # check allowed dims
-    if kind == 'hindcast':
-        is_in_list(dim, ['member', 'init'], 'dim')
-    elif kind == 'PM':
-        is_in_list(dim, ['member', 'init', ['init', 'member']], 'dim')
+        dim = ['init'] if kind == 'hindcast' else ['init', 'member']
```

Comment:
we could also prevent default keywords for comparison in this PR. should we @bradyrx?

Reply:
Let's just keep this as is for this PR. We can discuss the requirement of `dim` for `.verify()` funcs. I feel like it's very common to expect a reduction over `init` for users, but makes sense to expect them to know metric, alignment, etc. We can talk more about it